README.md

```diff
@@ -17,3 +17,8 @@ Also give a theoretical argument for why X could or could not be correct, based
 on the complexity of the general sorting problem we covered in class.
 
 Add your answers to this markdown file.
+
+Answer: 
+To verify this claim, I would take a list of n size and create every possible permutation of this list. Then, I would use this sorting algorithm to solve every permutation of the list and see which one had the worst case runtime. Now, I would repeat this strategy for multiple different sizes of lists with different elements. If all of these were solved in worst case time, and the time would grow linearly, then X's claim would be verified.  
+
+X could be correct if his algorith consistently grows linearly. This means when we look at multiple different sizes of worst case arrays, we will expect the runtime of the algorithm to grow linearly as well. If this fact does not prove to be true, then X's argument would prove to be false. There are some factors that we would have to take into account to prove if X's claim is correct. We know that theoretically, comparison-based sorting algorithms are restricted to at least $O(nlog n)$ time complexity because the function has to compare every element in the array at least once. The amount of comparisons after can be reduced to grow by log n, but it is theoretically impossible for a comparison-based sorting algorithm to get any more efficient than that. Therefore, X's sorting algorithm would have to be a non-comparison-based sorting algorithm to acheive this time complexity. Additionally, this claim would also have to take into account multiple different data types and test if all of them also grow linearly. Additionally, we would have to test the space complexity of the alogirthm. If the algorithm was extremely memory intensive, the algorithm could be completely impractical where we cannot possibly store the amount of data that the alogorithm requires or it causes significant slow down times to the point where the runtime does not grow linearly anymore. 
```

Comment:
So could the claim be correct or not? It might help to read the problem description again.